.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+See the [Scientific Python Developer Guide][spc-dev-intro] for a detailed
+description of best practices for developing scientific packages.
+
+[spc-dev-intro]: https://scientific-python-cookie.readthedocs.io/guide/intro
+
+# Quick development
+
+The fastest way to start with development is to use `nox``. This will set up a
+virtual environment for you to run all the checks and tests. `nox`` is pre-installed
+in the Codespaces environment.
```

Comment:
It is great that nox is pre-installed in the Codespaces environment, but we also want to allow external developers to know how to install it in their own machines.